host_core/mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule HostCore.MixProject do
       version: "0.20.0",
       elixir: "~> 1.12",
       start_permanent: Mix.env() == :prod,
-      compilers: [:rustler] ++ Mix.compilers(),
+      #   compilers: [:rustler] ++ Mix.compilers(),
```

Comment:
Is this an intentional change? I wasn't sure if this was left off to speed up dev time or something that should be removed entirely

Reply:
Yes, this is intentional. We were getting a warning that the compiler was deprecated and the folks in the Rustler channel in Elixir slack told me to remove it.